src/js/media-time-range.js

```diff
@@ -21,13 +21,9 @@ const template = document.createElement('template');
 template.innerHTML = `
   <style>
     :host {
-      --media-preview-background-color: rgba(20,20,30, .5);
```

Comment:
- Removing values with defaults here and relying on well-defined compositions now defined in `media-chrome-range`.
  - This changes some default values here, which can be overridden if desired.